packages/components/src/Notification/ToastNotification/types.ts

```diff
@@ -0,0 +1,33 @@
+import { NotificationType } from "../types"
+
+type Modify<T, R> = Omit<T, keyof R> & R
+
+export type ToastNotification = {
```

Comment:
I'm wary this may cause a conflict with the type generated for the `ToastNotification` component, but it looks like it was already like that so I suppose we leave it lest we break it. Something to keep in mind though.

Reply:
Felt a little ick, but hesitant to change the name since it was part of the original types